Yoast/wordpress-seo
# Remove content image fallback for Open Graph and Twitter

## Summary


This PR can be summarized in the following changelog entry:

* Removes the content and gallery image fallbacks for the twitter and open graph images in the indexables.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Facebook meta data should be enabled: `SEO` -> `Social` -> `Facebook` -> `Add Open Graph meta data` -> `Enabled`.
* Facebook meta data default should be cleared: `SEO` -> `Social` -> `Facebook` -> `Default settings` -> `Image URL` -> should be empty, or press `Clear Image` and save.
* Twitter meta data should be enabled: `SEO` -> `Social` -> `Twitter` -> `Add Twitter card meta data` -> `Enabled`.

### Test instructions on this branch
Follow the same steps as the reproduction steps on `feature/indexables-frontend` (see below). However, this time, the meta tag should **not** be in the source code.

### Reproduction steps on `feature/indexables-frontend`

#### Posts, content image
* Create a new post.
* Add an image to the post contents.
* Save the post as a draft.
* View the post on your website.
  * Check the post's source code, it should contain a meta tag like this. With `og:image` as the property and a link to the added image as content.
    ```html
     <meta property="og:image" content="https://yoast.com/app/uploads/2019/09/an-example-of-a-meta-description-on-yoast.com_.jpg">
    ```

#### Posts, gallery image
* Create a new post.
* Add a image gallery shortcode with an image to the post.
* Save the post as a draft.
* View the post on your website.
  * Check the post's source code, it should contain a meta tag like this. With `og:image` as the property and a link to the added image as content.
    ```html
     <meta property="og:image" content="https://yoast.com/app/uploads/2019/09/an-example-of-a-meta-description-on-yoast.com_.jpg">
    ```

#### Terms, content image
* Create a new category or tag.
* Add an image to the category's description.
* Save the category.
* View the category's archive page on your website.
  * Check the category's source code, it should contain a meta tag like this. With `og:image` as the property and a link to the added image as content.
    ```html
     <meta property="og:image" content="https://yoast.com/app/uploads/2019/09/an-example-of-a-meta-description-on-yoast.com_.jpg">
    ```

## UI changes

## Documentation

## Quality assurance


Fixes #14257
